src/components/chart/SleepChartDay.tsx

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useState } from 'react';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+  ChartDataLabels
+);
+
+export default function SleepChartDay(props: {
+  sleepCycleLineLists: any[],
+}) {
+
+  let options: any = {
+    responsive: true,
+    maintainAspectRatio: false,
+    plugins: {
+      title: {
+        display: false
+      },
+      legend: {
+        display: false
+      },
+      datalabels: {
+        display: false
+      }
+    },
+    scales: {
+      x: {
+        grid: {
+          display: false
+        },
+        ticks: {
+          font: {
+            size: 10,
+          },
+          color: '#ffffff',
+        }
+      },
+      y: {
+        stepSize: 400,
+        grid: {
+          display: false,
+        },
+        ticks: {
+          display: false
+        }
+      }
+    }
+  };
+
+  const [data, setData] = useState({
+    labels: [],
+    datasets: [
+      {
+        data: [],
+        backgroundColor: '#ff7644',
+        borderRadius: 6
+      },
+    ],
+  })
+
+  const getFormatTime = (dateString: string) => {
+    const date = new Date(dateString);
+    const hours = date.getHours();
+    const minutes = date.getMinutes();
+    return `${hours < 10 ? '0' + hours : hours}:${minutes < 10 ? '0' + minutes : minutes}`;
+  }
```

Comment:
ย้ายไปไว้ common ไหม

Reply:
ทุกอย่างดูดีแล้ว แต่ไปดูเรื่องที่มันส่ง record ไปให้ be ทุก ๆ 1 นาทีหน่อยว่า server หลังบ้านมันจะรองรับได้กี่คนแต่คิดว่าตัวเลขนี้น่าจะดีเเล้ว approved ให้ก่อนละกัน LGTM kubbbbb